docs/configuration/radio/device.mdx

```diff
@@ -47,6 +49,7 @@ This table shows the **default** values after selecting a preset. As always, ind
 | TAK            | Yes             | Optional       | Regular           | Yes            | No                  | Yes                   |
 | TAK_TRACKER    | Yes             | Optional       | Regular           | Yes            | No                  | Yes                   |
 | ROUTER         | No[^2]          | No             | High              | Yes            | Yes                 | Yes                   |
+| ROUTER_LATE    | Yes             | Yes            | Regular           | Yes            | No                  | Yes                   |
```

Comment:
I think we should set power consumption to "High" as it will rebroadcast as much as `ROUTER`/`REPEATER`, and it also has BLE/WiFi/Serial still enabled.

Reply:
Why have we left BLE and WiFi on for a router role?

Reply:
I think for a rooftop node that's still a valid use case.